### ISSUE TITLE
Kickstart won't work unless lazy installed first.

### ISSUE THREAD

**Issue by paulmfoster** [nvim-lua/kickstart.nvim#741]
## Describe the bug

After kickstart.nvim is installed, and nvim is run, errors appear to the effect that the lazy package is not found. Thus other packages don't install. That is, the lazy package will not install itself as it did in my prior config. However, if the lazy package is already installed in ~/.local/share/nvim, then everything works as promised. 

## To Reproduce

1. Remove contents of ~/.local/share/nvim.
2. Download zip of kickstart.nvim.
3. Unzip it.
4. Copy files into a blank ~/.config/nvim directory.
5. Run nvim
6. Note error message complaining about lack of lazy module.

## Desktop

- OS:  Linux Debian 12
- Terminal:  Alacritty

## Neovim Version

nvim version 0.9.1

**Comment by daveragos** [comment]
yeah i got the same error to, when i try to install nvim from scratch in my ubuntu 23.10 and does everything with https://www.youtube.com/watch?v=m8C0Cq9Uv9o&t=302s video , i got the same error that asks for lazy module

**Comment by dam9000** [comment]
Do you have `git` installed? It won't work without it.
Another thing to try is removing all nvim folders:
```
rm -r ~/.config/nvim/
rm -r ~/.local/share/nvim/
rm -r ~/.local/state/nvim/
rm -r ~/.cache/nvim/
```

**Comment by daveragos** [comment]
yeah i installed using the linux tar(before that i was using apt) and it kicksater but some plugins aren't installing

**Comment by dam9000** [comment]
lazy requires git, just install tat and the other dependencies as listed in the README

**Comment by paulmfoster** [comment]
On Mon, Mar 11, 2024 at 05:49:32AM -0700, Damjan 9000 wrote:

>    lazy requires git, just install tat and the other dependencies as
>    listed in the README

Git was already installed.

-- 
Paul M. Foster
Personal Blog: http://noferblatz.com
Company Site: http://quillandmouse.com
Software Projects: https://gitlab.com/paulmfoster

**Comment by paulmfoster** [comment]
On Mon, Mar 11, 2024 at 01:54:17AM -0700, Damjan 9000 wrote:

>    Do you have git installed? It won't work without it.
>    Another thing to try is removing all nvim folders:
> rm -r ~/.config/nvim/
> rm -r ~/.local/share/nvim/
> rm -r ~/.local/state/nvim/
> rm -r ~/.cache/nvim/

Git is already installed.

-- 
Paul M. Foster
Personal Blog: http://noferblatz.com
Company Site: http://quillandmouse.com
Software Projects: https://gitlab.com/paulmfoster

**Comment by dam9000** [comment]
@paulmfoster I tried it now in debian 12 and it works fine with nvim 0.9.5, perhaps try upgrading neovim. The kickstart checkhealth recommends version 0.9.4:
`lua/kickstart/health.lua`:
```
  if vim.version.cmp(vim.version(), { 0, 9, 4 }) >= 0 then
    vim.health.ok(string.format("Neovim version is: '%s'", verstr))
  else
    vim.health.error(string.format("Neovim out of date: '%s'. Upgrade to latest stable or nightly", verstr))
  end
```
Here is how I tried debian 12 and neovim 0.9.5:

(remove all the before mentioned nvim dirs)

```
distrobox create -i debian:12 debian12
distrobox enter debian12

cat /etc/debian_version
12.5

sudo apt update
sudo apt install make gcc ripgrep unzip git

echo "deb https://deb.debian.org/debian unstable main" | sudo tee -a /etc/apt/sources.list
sudo apt update
sudo install -t unstable neovim

nvim --version
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.0-beta3

git clone https://github.com/nvim-lua/kickstart.nvim.git ~/.config/nvim

nvim
```
and it works perfectly fine.

So I suspect it must be something in your environment, either user or system.

**Comment by feoh** [comment]
A thing I'm noticing: All the folks having issues here are using zip and tar and not git.

Crazy idea: Git clone rather than zip/tar?

**Comment by paulmfoster** [comment]
On Fri, Mar 15, 2024 at 07:47:46AM -0700, TJ DeVries wrote:

>    Has everyone in this thread done the suggestion of removing all the
>    nvim related folders? That can sometimes cause problems from a previous
>    nvim installation

Okay. I wiped out ~/.local/share/nvim and ~/.config/nvim. Then I cloned the
kickstart repo, moved its files to ~/.config/nvim. Then ran neovim, and all
went as planned.

The first time I tried this, I used the zip file rather than cloning the
repo, and that was a week or two ago, so the repo contents may have
changed. I did wipe out ~/.config/nvim and replaced it with the zip
contents. Can't recall whether I wiped out ~/.local/share/nvim or not. But
in any case, it didn't work.

Sure would be nice to know why leaving files in place (presumably in
~/.local/share/nvim) messes with things.

Paul

-- 
Paul M. Foster
Personal Blog: http://noferblatz.com
Company Site: http://quillandmouse.com
Software Projects: https://gitlab.com/paulmfoster
